ocs_ci/deployment/vmware.py

```diff
@@ -1626,6 +1626,15 @@ def destroy_cluster(self, log_level="DEBUG"):
             template_folder = get_infra_id(self.cluster_path)
         else:
             logger.warning("metadata.json file doesn't exist.")
+        vsphere = VSPHERE(
+            config.ENV_DATA["vsphere_server"],
+            config.ENV_DATA["vsphere_user"],
+            config.ENV_DATA["vsphere_password"],
+        )
+        all_vms = vsphere.get_vms_by_string(config.ENV_DATA["cluster_name"])
+        vsphere.stop_vms(all_vms)
+        for vm in all_vms:
```

Comment:
I think we should proceed with cluster teardown even in cases when disk removal fails. @shyRozen @vavuthu what do you think?

Reply:
@ebenahar We can do that but there could be situation where un-associated objects will be created, and if we will not be aware of the issue of disks not being deleted, we can end up with the situation we have today.
If we really want to proceed with the destroy we can think of a way of knowing that a problem occurred like mail at least. Maybe we can set config flag to fail it at the end of the destroy and then the destroy will continue but will notify failed and will raise awareness.

Reply:
an email sounds like a good idea. 
In addition, we can still consider running a user object cleanup job periodically, to collect all those leftovers